core/templates/components/state-directives/outcome-editor/outcome-editor.component.html

```diff
@@ -129,7 +129,7 @@
         name="$ctrl.editOutcomeForm.editDestForm"
         ng-submit="$ctrl.saveThisDestination()">
     <oppia-outcome-destination-editor [outcome]="$ctrl.outcome"
-                                      (get-changes)="getChanges()"
+                                      (get-changes)="$ctrl.getChanges()"
```

Comment:
Prior to this change, what kind of issue / bug did it result in?

Reply:
changed like earlier

Reply:
Hi @heyimShivam, I wasn't asking to undo this change, sorry if I wasn't clear earlier. I meant, why do we need this change -- is something broken without it?